packages/react/package.json

```diff
@@ -25,7 +25,8 @@
     "start:legacy": "yarn workspace @fluentui/public-docsite-resources start",
     "start-test": "just-scripts jest-watch",
     "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest -u"
+    "update-snapshots": "just-scripts jest -u",
+    "mf": "just-scripts mf"
```

Comment:
Can we...maybe find a different acronym? 🤣

Reply:
hahahah mf is everywhere :)